defold-spine/src/comp_spine_model.cpp

```diff
@@ -1163,6 +1163,12 @@ namespace dmSpine
             if (!track)
                 return dmGameObject::PROPERTY_RESULT_INVALID_INDEX;
 
+            if (!track->m_AnimationInstance)
+            {
+                dmLogError("Could not set playback rate since no animation is playing");
+                return dmGameObject::PROPERTY_RESULT_UNSUPPORTED_VALUE;
+            }
```

Comment:
Looking at the code, the logging isn't consistent with the other "errors" above. That is, it logs an error, whereas the other cases don't.

So, there will always be a trade off with more strings in the executable, but I think it may be worth it for cases where it's especially tricky to understand why it failed.

Reply:
I actually modelled it on the handling of the equivalent case above when setting PROP_CURSOR.
It is useful to be alerted to the error, in our case we don't intend to set playback_rate when no anim is playing, but we had to rewrite our state machine due to https://github.com/defold/extension-spine/issues/72 and it's not perfect.